ngraph/frontend/paddlepaddle/src/op/range.cpp

```diff
@@ -0,0 +1,41 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "range.hpp"
+#include <ngraph/opsets/opset6.hpp>
+
+namespace ngraph
+{
+    namespace frontend
+    {
+        namespace pdpd
+        {
+            namespace op
+            {
+                NamedOutputs range(const NodeContext& node)
+                {
+                    auto start = node.get_ng_input("Start");
+                    auto stop = node.get_ng_input("End");
+                    auto step = node.get_ng_input("Step");
+
+                    bool keep_dims = false;
+                    const auto axis = ngraph::opset6::Constant::create(element::i64, Shape{}, {0});
+                    auto start_scalar =
+                        std::make_shared<ngraph::opset6::ReduceMin>(start, axis, keep_dims);
```

Comment:
It looks like you using ReduceMin instead of Squeeze, if so please replace for Squeeze.

Reply:
@mangguo321 Could you please adjust by Squeeze ?